docs/docs/cookbook/cel.md

```diff
@@ -12,3 +12,10 @@ before the provided date. This is particularly useful when attached to a
 target.signing_time >= timestamp('2025-05-31T00:00:00Z')
 ```
 
+## Prevent users from disabling gatekeeper
+
+Create a signing ID rule for `platform:com.apple.spctl` and attach the following CEL program
+
+```clike
+['--global-disable', '--disable', '--remove'].exists(flag, flag in args) ? BLOCKLIST : ALLOWLIST
```

Comment:
```suggestion
['--master-disable', '--disable', '--remove'].exists(flag, flag in args) ? BLOCKLIST : ALLOWLIST
```

Reply:
I did't see --master-disable in the man page. But trying it seems to work seems like we need both.

```
NAME
     spctl – SecAssessment system policy security

SYNOPSIS
     spctl --assess [-t type] [-] file ...
     spctl --status
     spctl --global-enable
     spctl --global-disable | --disable-status
```